From: Per Andersson <avtobiff@gmail.com>
Date: Wed, 20 Mar 2013 00:02:01 +0100
Subject: Do not use pending gem in tests

---
 test/test_helper.rb |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -5,7 +5,7 @@
 require "#{File.expand_path(File.dirname(__FILE__))}/../lib/asciidoctor.rb"
 
 require 'nokogiri'
-require 'pending'
+#require 'pending'
 
 ENV['SUPPRESS_DEBUG'] ||= 'true'
 
