From: Guillaume Grossetie <ggrossetie@gmail.com>
Date: Tue, 16 Jun 2015 19:46:04 +0200
Subject: Load data from /usr/share/asciidoctor path

Bug-Debian: https://bugs.debian.org/788052
---
 lib/asciidoctor.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/asciidoctor.rb b/lib/asciidoctor.rb
index 1c684ce..f3736fa 100644
--- a/lib/asciidoctor.rb
+++ b/lib/asciidoctor.rb
@@ -182,7 +182,7 @@ def define key, value
   LIB_DIR = ::File.join ROOT_DIR, 'lib'
 
   # The absolute data directory of the Asciidoctor RubyGem
-  DATA_DIR = ::File.join ROOT_DIR, 'data'
+  DATA_DIR = ::File.exist?(asciidoctor_lib_path = ::File.join(ROOT_DIR, 'data')) ? asciidoctor_lib_path : ::File.join(ROOT_DIR, '../../share/ruby-asciidoctor')
 
   # The user's home directory, as best we can determine it
   # IMPORTANT this rescue is required for running Asciidoctor on GitHub.com
