Description: remove tests requiring network access
Author: Jérémy Bobbio <lunar@debian.org>
Origin: vendor
Forwarded: not-needed

diff --git a/test/reader_test.rb b/test/reader_test.rb
index abc708a..5750437 100644
--- a/test/reader_test.rb
+++ b/test/reader_test.rb
@@ -499,17 +499,6 @@ include::fixtures/no-such-file.ad[]
         end
       end
   
-      test 'include macro can retrieve data from uri' do
-        input = <<-EOS
-....
-include::https://raw.github.com/asciidoctor/asciidoctor/master/LICENSE[]
-....
-        EOS
-  
-        output = render_embedded_string input, :safe => :safe, :attributes => {'allow-uri-read' => ''}
-        assert_match(/MIT/, output)
-      end
-  
       test 'inaccessible uri referenced by include macro does not crash processor' do
         input = <<-EOS
 ....
